### PR TITLE
[13.0][MIG] oca_event_badge : Migration to 13.0

### PR DESCRIPTION
**Pull request by manuelcalerosolis**
This is the migration to 13.0 of the module oca_event_badge

@Tecanativa TT23195

### CODE REVIEW

**Review by ernestotejeda — COMMENTED** [review]
Thanks @manuelcalerosolis 

Travis is red.
uff, I can't any migration commit here ?!!

File: oca_event_badge/static/src/css/event_event_template.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/* Copyright 2018 ACSONE SA/NV
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+.badge_logo {
+    max-height:1cm;
```
Comment (Tardo) [inline oca_event_badge/static/src/css/event_event_template.scss]: Use 'cm' to display in screens are not recommended.
See https://www.w3.org/Style/Examples/007/units.en.html

File: oca_event_badge/views/event_event_template_asset_report.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 ACSONE SA/NV
+     Copyright 2019 Tecnativa - Cristina Martin R.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <template id="assets_common" inherit_id="web.report_assets_common">
+        <xpath expr="." position="inside">
+          <link href="/oca_event_badge/static/src/css/event_event_template.scss" rel="stylesheet" type="text/scss"/>
```
Comment (Tardo) [inline oca_event_badge/views/event_event_template_asset_report.xml]:
```suggestion
          <link href="/oca_event_badge/static/src/css/event_event_template.scss" rel="stylesheet" type="text/css"/>
```

File: oca_event_badge/views/event_event_templates.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+    <template id="oca_event_badge_registration_report_logo_template" inherit_id="event.event_registration_report_template_badge">
+        <xpath expr="//div[@id='o_event_name']" position="inside">
+            <div class="row">
+                <div t-if="bool(len(o.partner_id.category_id))" class="col-12 text-center mt16" t-ignore="true">
+                    <span class="text-center" t-foreach="o.partner_id.category_id" t-as="val">
+                        <t t-if="val.name == 'board'">
+                            <img src="/oca_event_badge/static/src/img/oca_board.png" class="badge_logo"/>
+                        </t>
+                        <t t-if="val.name == 'delegate'">
```
Comment (Tardo) [inline oca_event_badge/views/event_event_templates.xml]:
```suggestion
                        <t t-elif="val.name == 'delegate'">
```

File: oca_event_badge/views/event_event_templates.xml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                            <img src="/oca_event_badge/static/src/img/oca_delegate.png" class="badge_logo"/>
+                        </t>
+                        <t t-if="val.name == 'member'">
```
Comment (Tardo) [inline oca_event_badge/views/event_event_templates.xml]:
```suggestion
                        <t t-elif="val.name == 'member'">
```

File: oca_event_badge/views/event_event_templates.xml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+                </div>
+            </div>
+
+        </xpath>
+    </template>
+
+    <template id="oca_event_badge_registration_report_logo_template" inherit_id="event.event_registration_report_template_badge">
+        <xpath expr="//div[@id='o_event_name']" position="inside">
+            <div class="row">
+                <div t-if="bool(len(o.partner_id.category_id))" class="col-12 text-center mt16" t-ignore="true">
```
Comment (Tardo) [inline oca_event_badge/views/event_event_templates.xml]:
```suggestion
                <div t-if="o.partner_id.category_id" class="col-12 text-center mt16" t-ignore="true">
```

**Review by Tardo — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
@Tardo please take over this and finish it.

File: oca_event_badge/static/src/css/event_event_template.scss (hunk 1 above)
Comment (manuelcalerosolis) [reply oca_event_badge/static/src/css/event_event_template.scss]: I would gladly change it but, I don't have write permissions already in this repo. Just out of curiosity, what unit of measure would you recommend?

Comment (pedrobaeza) [reply oca_event_badge/static/src/css/event_event_template.scss]: @Tardo I would say this is for printing the badge on paper, so it's not on screen.

**Comment by pedrobaeza** [comment]
@CarlosRoca13, as @Tardo is with other things, can you please retake this?

**Comment by CarlosRoca13** [comment]
@pedrobaeza Sure!

**Comment by pedrobaeza** [comment]
Please avoid merge commits. Do rebase instead.

**Review by ernestotejeda — COMMENTED** [review]
For me, 
Functionally OK.
Code OK.

Attend the @pedrobaeza comment. Leave the pre-commit changes in a separate commit and the migration changes in another commit.

**Comment by CarlosRoca13** [comment]
@ernestotejeda change done!

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**